installer/dockerfile/controller-manager/Dockerfile

```diff
@@ -12,8 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+FROM golang:1.16.5 AS builder
```

Comment:
There are three places where golang mirrors are used. If I want to update the version of go that compiles volcano, I need to update three places. Is there any room for optimization here?